charts/cloudserver-front/values.yaml

```diff
@@ -29,6 +29,12 @@ image:
   tag: 0.2.1
   pullPolicy: IfNotPresent
 
+proxy:
+  # If you want to use an http proxy, remove the curly braces
+  # after 'http' and/or 'https', and add the respective endpoints.
+  http: {}
```

Comment:
If an empty string is used, the variables would be exported as empty strings. Also {{- if .Values.proxy.https }} will be true for an empty string. Commenting them out will cause a templating error because ".Values.proxy.https" won't exist. These are the reasons why I initially went with the other solution.